og/lint/action.yml

```diff
@@ -42,20 +42,24 @@ runs:
           files_to_format = edited_py_files - ignore_files
 
           if files_to_format := " ".join(map(str, files_to_format)):
-            print("Running black...")
-            subprocess.run(f"black {files_to_format}", shell=True)
-            print("Running isort...")
-            subprocess.run(f"isort --profile=black {files_to_format}", shell=True)
-            print("Running docformatter")
-            if (
+            try:
+              print("Running black...", flush=True)
+              subprocess.run(f"black {files_to_format}", shell=True)
+              print("Running isort...", flush=True)
+              subprocess.run(f"isort --profile=black {files_to_format}", shell=True)
+              print("Running docformatter", flush=True)
+              if (
+                subprocess.run(
+                  f"docformatter --recursive --in-place --wrap-summaries 88 --wrap-descriptions 88 {files_to_format}",
+                  shell=True,
+                ).returncode
+                == 3
+              ):
+                print("Docformatter raised status 3", flush=True)
+              print("Running flake8", flush=True)
               subprocess.run(
-                f"docformatter --recursive --in-place --wrap-summaries 88 --wrap-descriptions 88 {files_to_format}",
-                shell=True,
-              ).returncode
-              == 3
-            ):
-              print("Docformatter raised status 3")
-            print("Running flake8")
-            subprocess.run(
-              f"flake8 --ignore=E203,E501,W503 {files_to_format}", shell=True, check=True
-            )
+                f"flake8 --ignore=E203,E226,E227,E501,W503, {files_to_format}", shell=True, check=True
+              )
+            except Exception:
+              print("Lint action finished with an exception:", flush=True)
```

Comment:
fyi I put this here mainly to get stdout flushed as sometimes the gh action would not display everything.